hibernate-core/src/main/java/org/hibernate/property/access/spi/GetterFieldImpl.java

```diff
@@ -98,30 +100,30 @@ public Member getMember() {
 
 	@Override
 	public String getMethodName() {
-		return null;
+		return getterMethod != null ? getterMethod.getName() : null;
 	}
 
 	@Override
 	public Method getMethod() {
-		return null;
+		return getterMethod;
 	}
```

Comment:
I've checked the usage:
1.  getMethod is used just for ``Modifier.isFinal( method.getModifiers() )`` 
2. getMethodName -  ``getterNames[0] = getter.getMethodName();``